docs/sources/project/get-help.md

```diff
@@ -2,103 +2,127 @@ page_title: Where to chat or get help
 page_description: Describes Docker's communication channels
 page_keywords: IRC, Google group, Twitter, blog, Stackoverflow
 
+<style>
+/* @TODO add 'no-zebra' table-style to the docs-base stylesheet */
+/* Table without "zebra" striping */
+.content-body table.no-zebra tr {
+  background-color: transparent;
+}
+</style>
+
 # Where to chat or get help
 
-There are several communications channels you can use to chat with Docker community members and developers. 
+There are several communications channels you can use to chat with Docker
+community members and developers.
 
-<style type="text/css">
-.tg  {border-collapse:collapse;border-spacing:0;text-align: left;}
-.tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;vertical-align:top;}
-.tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
-</style>
-<table class="tg">
+<table>
```

Comment:
I'd prefer to leave my table styling in the document until your table styling merge hits the external documentation. I checked quickly and didn't see it here: https://github.com/docker/docker/commits/docs was it accepted and merged in master?